src/day-1-toy/lists.py

```diff
@@ -8,22 +8,25 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # [command here]
-print(x)
+print(x.append(4))
 
```

Comment:
I think these might work better if you perform the function first and then print afterward. Like x.append(4), then print(x). Otherwise, looking good!